beacon-chain/rpc/eth/debug/debug.go

```diff
@@ -50,12 +53,61 @@ func (ds *Server) GetBeaconStateSSZ(ctx context.Context, req *ethpbv1.StateReque
 	return &ethpbv1.BeaconStateSSZResponse{Data: sszState}, nil
 }
 
-func (ds *Server) GetBeaconStateV2(ctx context.Context, request *ethpbv2.StateRequestV2) (*ethpbv2.BeaconStateResponseV2, error) {
-	panic("implement me")
+// GetBeaconStateV2 returns the full beacon state for a given state ID.
+func (ds *Server) GetBeaconStateV2(ctx context.Context, req *ethpbv2.StateRequestV2) (*ethpbv2.BeaconStateResponseV2, error) {
+	ctx, span := trace.StartSpan(ctx, "beaconv1.GetBeaconStateV2")
```

Comment:
```suggestion
	ctx, span := trace.StartSpan(ctx, "beaconV1.GetBeaconStateV2")
```

Better? or should we eliminate v1 all at once? Seems weird to have a v1 and v2

Reply:
we should eliminate server versions

Reply:
and it should be called `debug` not `beacon` here